### ISSUE TITLE
Fail build with run-android when updating from 2.0.7 to 2.0.8

### ISSUE THREAD

**Issue by yenda** [realm/realm-js#1506]
## Goals

I am trying to run a react-native app that was previously working with 2.0.7

## Expected Results

react-native run-android works

## Actual Results

react-native run-android fails

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> Cannot evaluate module realm : Configuration with name 'default' not found.
```

## Code Sample

This is the project: https://github.com/status-im/status-react

## Version of Realm and Tooling

- Realm JS SDK Version: realm 2.0.8
- Node or React Native: - Client OS & Version: 0.43.4
- Which debugger for React Native: None

**Comment by chlela** [comment]
I noticed that ./node_modules/realm/android folder is missing in version 2.0.8

**Comment by mdumrauf** [comment]
This made me lost a lot of time the other day. @realm-admin why don't you use semver? 2.0.8 is supposed to be a  patch version, not a breaking changes one!

**Comment by kneth** [comment]
Sorry, I had an issue with RN Android during release so I had to disable it. I plan to do a 2.0.9 release to with RN Android enabled.

**Comment by kneth** [comment]
@mdumrauf @chlela @yenda I have released 2.0.9 - with RN Android support.
